sdk/js-cloud-server/package.json

```diff
@@ -8,7 +8,11 @@
     },
     "dependencies": {
         "@devcycle/types": "^1.1.16",
-        "lodash": "^4.17.21"
+        "cross-fetch": "^3.1.8",
+        "fetch-retry": "^5.0.3",
+        "lodash": "^4.17.21",
+        "msw": "^2.0.0"
```

Comment:
I think this should be a devDep

Reply:
maybe we can even remove it as we probably don't want folks installing mock service worker.